test/Makefile

```diff
@@ -11,3 +12,8 @@ fake-next-minor-rpm:
 	MICROSHIFT_VERSION="$(MAJOR).$(NEXT_MINOR).$(PATCH)_fake_next_minor" \
 	RPMBUILD_DIR=$(RPMBUILD_DIR) \
 		$(MAKE) -C $(REPO) rpm
+
+.PHONY: robotidy
```

Comment:
How about moving logic of `scripts/verify/verify-rf.sh` here?

Reply:
Good point. We already have that functionality, so let's reuse it.